pom.xml

```diff
@@ -379,6 +380,31 @@
                 <artifactId>maven-resources-plugin</artifactId>
                 <version>2.7</version>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-checkstyle-plugin</artifactId>
+                <version>2.17</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>check</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <violationSeverity>error</violationSeverity> <!-- change to 'warning' to enforce strictly -->
+                    <violationIgnore>indentation</violationIgnore>
+                </configuration>
+            </plugin>
+            <plugin>
+                <!--
+                To apply: mvn googleformatter:format
+                To repair indentation (BSD/Mac sed): git status -s | sed -En 's/^ M (src.*\.java)$/\1/p' | xargs sed -Ei '' 's/^( *)/\1\1/'
+                -->
+                <groupId>com.theoryinpractise</groupId>
+                <artifactId>googleformatter-maven-plugin</artifactId>
+                <version>1.0.6</version>
```

Comment:
The plugin [allows](https://github.com/talios/googleformatter-maven-plugin/blob/master/src/main/java/com/theoryinpractise/googleformatter/GoogleFormatterMojo.java#L74) one to configure [AOSP](https://github.com/google/google-java-format/blob/master/core/src/main/java/com/google/googlejavaformat/java/JavaFormatterOptions.java#L40) style, which uses four spaces.